cli/cli.go

```diff
@@ -24,12 +19,16 @@ func InitArgs() (Args, bool) {
 	args := Args{}
 	isMissingArg := false
 
-	args.Tags = setFlag("tags", "")
-	args.Dir = setFlag("dir", ".")
-	args.IsSkipTerratagFiles = booleanFlag("skipTerratagFiles", true)
-	args.Filter = setFlag("filter", ".*")
-	args.Verbose = booleanFlag("verbose", false)
-	args.Rename = booleanFlag("rename", true)
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```

Comment:
Could you please extract `os.Args[0]` and `os.Args[1:]` into meaningful variables and use those instead? 
i.e
```
programName := os.Args[0]
programArgs := os.Args[1:]

Reply:
Good catch, thank you! Updated also the test file just to have consistency.